website/content/partials/known-issues/perf-standbys-revert-to-standby.mdx

```diff
@@ -0,0 +1,48 @@
+### Performance Standbys revert to Standby mode on unseal
+
+#### Affected versions
+
+- 1.15.8
```

Comment:
```suggestion
- 1.14.12
- 1.15.8
```

Reply:
I think we need to add 1.14.12 throughout, and also add the partial to the 1.14 upgrade notes.

Reply:
Thanks, @mladlow! I pushed up some fixes with your feedback incorporated (here and below).